requirements/test.txt

```diff
@@ -1,3 +1,5 @@
 pytest>=6.0
 pytest-cov>=2.10
 codecov>=2.1
+ete3>=3.1
+six>=1.15
```

Comment:
The dependencies should only be listed in `extra` I think. The `importorskip` in the ete3 test file should be sufficient to handle this situation correctly.